src/visor/visor.go

```diff
@@ -159,17 +161,73 @@ func (c Config) Verify() error {
 	return nil
 }
 
+// historyer is the interface that provides methods for accessing history data that are parsed from blockchain.
+type historyer interface {
```

Comment:
make this public too, Historyer

Reply:
The `historyer` interface is only used in the visor package, so don't have to make it public,. And go prefer expose struct instead of interface, expose interface only if the package have public methods or functions that have the interface as parameter, otherwise should keep it in the package.